apps/site/pages/en/about/previous-releases.mdx

```diff
@@ -50,3 +50,4 @@ Community installation methods included on the self-service download page (/down
 - **Broad OS Support:** Cannot be limited to a subset of OS distributions or versions.
   - For example, an installation method claiming compatibility with “Windows” must function on “Windows 10”, “Windows 11”, and all their editions (including server versions).
   - Similarly, an installation method claiming compatibility with “Linux” must be installable on all major Linux distributions, not just a specific subset. It cannot rely on distribution-specific package managers like `apt` or `dnf`.
+- **Free and Open Source:** Must be free to use and open source, must not be sold as a commercial product, and must not be a paid service.
```

Comment:
Maybe rather clarifying it cannot be a product from a for-profit? I know these two are distinct. But all options we have fit the “not products from a company” criteria. Except of course Docker, which is an exception as it is a 1st party installation method.

Reply:
```suggestion
- **Free and Open Source:** Must be free to use and open source, must not be sold as a commercial product or developed by a for-profit entity, and must not be a paid service.
```
Does this cover what you're getting at?